### PR TITLE
en: add faq about configUpdateStrategy

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added, or deleted? (Required)

- Add a faq about configratuion update stragety.
- Add description for `configUpdateStrategy: RollingUpdate`.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1718
- Other reference link(s):

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ Here are the formats of the parameters:
 
 #### configUpdateStrategy
 
-It is recommended that you configure `spec.configUpdateStrategy: RollingUpdate` to enable automatic update of configurations. This way, every time the configuration is updated, all components are rolling updated automatically, and the modified configuration is applied to the cluster.
+The default value of the `spec.configUpdateStrategy` field is `InPlace`, which means that when you modify `config` of a component, you need to manually trigger a rolling update so that the new configuration is applied to the cluster.
```
Comment (qiancai) [inline en/configure-a-tidb-cluster.md:54]:
```suggestion
The default value of the `spec.configUpdateStrategy` field is `InPlace`, which means that when you modify `config` of a component, you need to manually trigger a rolling update to apply the new configurations to the cluster.
```

File: en/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -51,7 +51,9 @@ Here are the formats of the parameters:
...
+
+It is recommended that you configure `spec.configUpdateStrategy: RollingUpdate` to enable automatic update of configurations. This way, every time the `config` of a component is updated, the component is automatically rolling updated, and the modified configuration is applied to the cluster.
```
Comment (qiancai) [inline en/configure-a-tidb-cluster.md:56]:
```suggestion
It is recommended that you configure `spec.configUpdateStrategy: RollingUpdate` to enable automatic update of configurations. In this way, every time the `config` of a component is updated, TiDB Operator automatically triggers a rolling update for the component and applies the modified configuration to the cluster.
```

File: en/faq.md [hunk 3]
```diff
@@ -137,4 +137,8 @@ After you execute the `kubectl get tc` command, if the output shows that the **R
 * Scaling
 * Any Pod of PD, TiDB, TiKV, or TiFlash is not Ready
 
-To check whether a TiDB Cluster is unavailable, you can try connecting to TiDB. If the connection fails, it means that the corresponding TiDBCluster is unavailable.
\ No newline at end of file
+To check whether a TiDB cluster is unavailable, you can try connecting to TiDB. If the connection fails, it means that the corresponding TiDBCluster is unavailable.
+
+## After modifying the configuration of a component, why does the new configuration not take effect?
```
Comment (qiancai) [inline en/faq.md:142]:
```suggestion
## After the configuration of a component is modified, why does the new configuration not take effect?
```

File: en/configure-a-tidb-cluster.md (hunk 2 above)
Comment (qiancai) [reply en/configure-a-tidb-cluster.md:56]: 如果保持不改， "is rolling updated"，里 rolling 的词性可能不太准确

**Review by qiancai — COMMENTED** [review]
@ran-huang I've completed the review and added some comments. PTAL.

**Review by qiancai — APPROVED** [review]

**Pull request merged**